_events/talks/event-010.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Towards Knowledge Graphs of Research Software metadata"
+authors:
+    - &speaker name:  Daniel Garijo
+      bio: Information Sciences Institute, USC
+      email: dgarijo@isi.edu
+      orcid: 0000-0003-0454-7145
+      is_speaker: true
+    - name:  Yolanda Gil
+      bio: Information Sciences Institute, USC
```

Comment:
Great you checked, that the affiliation needs to be adapted.

I think this will stay a manual adaptation of information. At least I currently don't see something simple to recognise, that *USC* should be appended to each of the affiliations.